### DISCUSSION TITLE
Namechange MQTT on HomeAssistant from (2023-08 to 2024-02)

### DISCUSSION THREAD

**Discussion by fugazzy** [emsesp/EMS-ESP32#1244] [General]
Hello, I am using EMS-ESP on my HomeAssistant. Since latest update to 2023-08 I get a warning about the naming of my MQTT-Sensors for this.
Liste der betroffenen Entitäten:

- binary_sensor.system_status
- sensor.ems_esp_status
- binary_sensor.ems_esp_status

I currenty use this settings:
![IMG_2270](https://github.com/emsesp/EMS-ESP32/assets/20287362/9a84b9e0-e94b-438b-b8dc-eb28277e64aa)

Is there a way to change the names without loosing existing data?

Also the Grafana dashboard does not work for me. I assume that is also because of the naming of the sensors?

**Comment by bbqkees** [comment]
There is already an issue about this topic:
https://github.com/emsesp/EMS-ESP32/issues/1243

Currently it's only a problem for 2 status sensors.
